src/__tests__/opportunities.int.test.ts

```diff
@@ -106,15 +81,9 @@ describe('/opportunities', () => {
 		});
 
 		it('returns 404 when id is not found', async () => {
-			await db.query(`
-        INSERT INTO opportunities (
-          title,
-          created_at
-        )
-        VALUES
-          ( 'This definitely should not be returned', '2525-01-02T00:00:06Z' ),
-          ( 'This surely also should not be returned', '2525-01-02T00:00:07Z' );
-      `);
+			await createOpportunity({
+				title: 'This definitely should not be returned',
```

Comment:
To clarify, we create this so that there is an opportunity in the database, just not the one with our specified id?

Reply:
@hminsky2002 yeah -- I do think this makes the test a little more robust